src/runtime_src/doc/toc/build.rst

```diff
@@ -19,40 +19,47 @@ Build the runtime
 
 ::
 
-   cd XRT/build
+   cd build
    ./build.sh
 
 ``build.sh`` script builds for both Debug and Release profiles.  On RHEL/CentOS, if ``build.sh`` was accidentally run prior to enabling the devtoolset, then it is necessary to clean stale files makefiles by running ``build.sh clean`` prior to the next build.
 
 Build RPM package on RHEL/CentOS or DEB package on Ubuntu
 .........................................................
 
-::
+The package is actually automatically built for the ``Release``
+version but not for the ``Debug`` version::
 
-   cd XRT/build/Release
+   cd build/Release
+   make package
+   cd ../Debug
    make package
 
 Install the XRT RPM package
 ...........................
 
-::
+Install with (the actual package name might differ) ::
 
    yum reinstall ./XRT-2.1.0-Linux.rpm
 
 Install the XRT DEB package
 ...........................
 
-::
+Install with (the actual package name might differ) ::
 
-   apt install --reinstall ./XRT-2.1.0-Linux.deb
+   sudo apt install --reinstall ./xrt_201830.2.1.0_18.10.deb
 
 XRT Documentation
 ~~~~~~~~~~~~~~~~~
 
-XRT Documentation can be built automatically using Sphinx doc builder together with Linux kernel based kernel-doc utility.
+XRT Documentation can be built automatically using Sphinx doc builder
+together with Linux kernel based ``kernel-doc`` utility.
 
-::
+To compile and install the documentation into the ``doc`` directory at
+the top of the repository::
 
-   cd XRT/src/runtime_src/doc/
-   make
-   firefox html/index.html
+   cd src/runtime_src/doc
+   cmake .
```

Comment:
.I would recommend running cmake only in temporary directories, e.g. 
% mkdir build
% cd build
% cmake ..
Makes it much easier to clean up.

Reply:
Yes, it would be great, even if `git clean` can help.
But it would require changing `src/runtime_src/doc/CMakeLists.txt` to make it work...
So just keep this PR low-profile with only non-functional change and leave some work for the future generations. :-)

Reply:
Works for me in a separate build directory.